HighwayPatrol.cpp

```diff
@@ -28,7 +31,7 @@ void HighwayPatrol::pullOver( Vehicle* v, bool willArrest, Highway* h )
     if( willArrest )
     {
         //print the vehicle type in this std::cout between "THE [" and "] PULL". 
-        std::cout << name << ": YOU IN THE [ " << " ] PULL OVER AND SHOW YOUR HANDS" << std::endl;
+        std::cout << name << ": YOU IN THE [ " << v->name <<  " ] PULL OVER AND SHOW YOUR HANDS" << std::endl;
```

Comment:
`v->name` is not the `vehicle type`.   The purpose of this task in this function is for you to demonstrate knowledge of how to cast across the inheritance hierarchy to determine an object's type correctly.
- [ ] revise